frontend/src/reducer/user.js

```diff
@@ -46,7 +46,11 @@ export const defaultUser = {
 export default function user (state = defaultUser, action) {
   switch (action.type) {
     case `${SET}/${USER_CONNECTED}`:
-      return { ...state, ...serialize(action.user, serializeUserProps) }
+      return {
+        ...state,
+        ...serialize(action.user, serializeUserProps),
+        lang: action.user.lang ? action.user.lang : state.lang
```

Comment:
This can be written more concisely:

```suggestion
        lang: action.user.lang || state.lang
```

But don't necessarily bother changing this, this is mostly a matter of taste and it's not really important.